### PR TITLE
DOC: DJM doc edits

### PR DESCRIPTION
**Pull request by darienmorrow**

### CODE REVIEW

**Comment by darienmorrow** [comment]
The LHS of the docs page does not link to "Writing a New From Function", is this desired?

**Comment by darienmorrow** [comment]
http://wright.tools/en/latest/write_from_function.html#write-from-function
> A few simple validation checks can be performed. If it is not possible to read a data object, it should raise a WrightTools exception. See :mod:`~WrightTools.exceptions`.
Should this link to somewhere?

**Comment by darienmorrow** [comment]
http://wright.tools/en/latest/write_from_function.html#write-from-function
> The function should have a docstring that documents all parameters. The summary line should tell about the source of the data. Feel free to add additional information in the body of the docstring, where appropriate. Check out the existing examples for formatting, such as the example from from_JASCO().

We use numpy style docstrings, right? Should we provide a link? e.g. https://numpydoc.readthedocs.io/en/latest/format.html

File: docs/artists.rst [hunk 1]
```diff
@@ -186,7 +186,7 @@ First, the full code and the image it creates:
 Layout
 ^^^^^^
 
-``WrightTools`` defines a handy function, :meth:`~WrightTools.artists.create_figure`, for easily and flexibly making complicated figures.
+``WrightTools`` defines a handy function, :meth:`~WrightTools.artists.create_figure`, to easily and flexibly making complicated figures.
```
Comment (ksunden) [inline docs/artists.rst]: either for...making or to...make, not to...making

**Comment by ksunden** [comment]
> http://wright.tools/en/latest/collection.html "We discuss data objects in the next section." The LHS has collection coming after data. Is this desired?

This should probably be fixed, I think I like the order it is in, and this text should be updated



> The LHS of the docs page does not link to "Writing a New From Function", is this desired?

It is linked to in `Contributing`, I personally liked it in the side bar, but @untzag had stronger opinions.



> http://wright.tools/en/latest/write_from_function.html#write-from-function
> 
> > A few simple validation checks can be performed. If it is not possible to read a data object, it should raise a WrightTools exception. See :mod:`~WrightTools.exceptions`.
> 
> Should this link to somewhere?

That is a link.



> http://wright.tools/en/latest/write_from_function.html#write-from-function
> 
> > The function should have a docstring that documents all parameters. The summary line should tell about the source of the data. Feel free to add additional information in the body of the docstring, where appropriate. Check out the existing examples for formatting, such as the example from from_JASCO().
> 
> We use numpy style docstrings, right? Should we provide a link? e.g. https://numpydoc.readthedocs.io/en/latest/format.html
> Note: we reference numpy docstrings in the 'contributing' section.

I could go either way, We do use numpy style, and do link to it in contributing, but sometimes it's easier to just show an example when it's this similar and leave it at that.

**Comment by darienmorrow** [comment]
@ksunden On my machine, `exceptions` is not a link.

**Comment by ksunden** [comment]
@darienmorrow Ahh, it is a link on stable, but not latest, for some reason we don't have that module level link now...

**Pull request merged**